README.md

```diff
@@ -454,6 +455,75 @@ and this record will be written to the specified index (`logstash-2014.12.19`) r
 
 Similar to `target_index_key` config, find the type name to write to in the record under this key (or nested record). If key not found in record - fallback to `type_name` (default "fluentd").
 
+### dynamic_target_index
```

Comment:
I feel that `dynamic_target_index` naming is inappropriate and ambiguous. 
ES plugin already handles `target_index{ dynamically with built-in placeholders.

Instead, I want to use `affinity` word to represent for "data affinity" or something in this case:

* `enable_target_index_affinity`
* `enable_affinity_by_id`

Reply:
Ok, thanks a lot for comments. Yes was not sure what name would be good for this feature. I can re-name it to target_index_affinity as you suggested.